src/Driver/WpcliDriver.php

```diff
@@ -111,25 +123,19 @@ public function wpcli($command, $subcommand, $raw_arguments = [])
             $config = sprintf('--path=%s --url=%s', escapeshellarg($this->path), escapeshellarg($this->url));
         }
 
-        // Support Windows.
-        if (DIRECTORY_SEPARATOR === '\\') {
-            $binary = 'wp.bat';
-        } else {
-            $binary = 'wp';
-        }
-
-        // Adjust bootstrap path, dependant on if WordHat is a Composer dependancy, or being run directly.
-        $package = '/vendor/paulgibbs/behat-wordpress-extension';
-        if ($package === substr(getcwd(), -strlen($package), strlen($package))) {
-            $package = '';
-        }
+        //Absolute path to the WordHat src directory
+        $src = dirname( __DIR__ );
 
         $cmd_output = [];
         $exit_code  = 0;
-        $wpcli_args = sprintf('--no-color --require=%1$s%2$s/src/WpcliLogger.php', getcwd(), $package);
+        $wpcli_args = sprintf('--no-color --require=%1$s/WpcliLogger.php', $src);
```

Comment:
If `$src` is only used one, please move the function call into the `sprintf`. I don't think that the line is going to get harder to read by doing so.